app/src/main/res/values/strings.xml

```diff
@@ -144,6 +144,8 @@
     <string name="online_game_buzzer_title">Online Buzzer</string>
     <string name="join_room">Join party room</string>
     <string name="create_room">Create party room</string>
+    <string name="game_setup_internet_switch_on">Internet is on</string>
+    <string name="game_setup_internet_switch_off">Internet is off</string>
```

Comment:
As written in the wiki, consider using string name that respect string convention and put them at the top of the document in the corresponding part. (see here https://github.com/MaximeZmt/SDP_2022-Vibester/wiki)
Meaning
<!-- gameSetup -->
gameSetup_internetSwitchOn
gameSetup_internetSwitchOff

Reply:
Understood, will be in the next commit soon